packages/test/test-service-load/testConfig.json

```diff
@@ -28,18 +28,21 @@
 			"optionOverrides": {
 				"odsp": {
 					"configurations": {
-						"Fluid.ContainerRuntime.Test.SummaryStateUpdateMethod": ["restart", "default"]
+						"Fluid.ContainerRuntime.Test.SummaryStateUpdateMethodV2": ["restart", "default"],
+						"Fluid.ContainerRuntime.Test.ValidateSummaryBeforeUpload": "true"
```

Comment:
We should have both true and false here because its not enabled by default.

Reply:
Oh, right.  I was thinking the pairwise thing would do present / absent.  I'll look again

Reply:
You will have to do [true, false] or [true, false, undefined]